packages/expect-utils/src/utils.ts

```diff
@@ -178,7 +192,12 @@ export const iterableEquality = (
   bStack.push(b);
 
   const iterableEqualityWithStack = (a: any, b: any) =>
-    iterableEquality(a, b, [...aStack], [...bStack]);
+    iterableEquality(a, b, [...customTesters], [...aStack], [...bStack]);
+
+  customTesters = [
```

Comment:
name of variable does not match similar code bellow, where its' `filteredCustomTesters `

Reply:
Alrighty. I'll try to make it more similar. I need to sit down and think through this recursive tester bit a little more as well.

Reply:
Updated the name to filteredCustomTesters